src/sdk.ts

```diff
@@ -439,6 +432,7 @@ export class OpenSeaSDK {
     paymentTokenAddress?: string;
     buyerAddress?: string;
   }): Promise<OrderV2> {
+    //TODO: Make this function multichain compatible
```

Comment:
Making changes to the createBuy/createSellOrder to support multichain will have cascading changes so holding off until a separate PR